src/shared/lib/comparison/ComparisonStore.ts

```diff
@@ -1208,6 +1208,24 @@ export default abstract class ComparisonStore extends AnalysisStore
         },
     });
 
+    readonly genesSortedByAlterationFrequency = remoteData<string[]>({
+        await: () => [
+            this.alterationsEnrichmentData,
+            this.alterationsEnrichmentAnalysisGroups,
+        ],
+        invoke: async () => {
+            const alterationRowData: AlterationEnrichmentRow[] = getAlterationRowData(
+                this.alterationsEnrichmentData.result!,
+                this.resultsViewStore
+                    ? this.resultsViewStore.hugoGeneSymbols
+                    : [],
+                this.alterationsEnrichmentAnalysisGroups.result!
+            );
```

Comment:
Looks like we are already computing this part in the [@computed get alterationEnrichmentRowData](https://github.com/cBioPortal/cbioportal-frontend/blob/6ffbe5e2a41ce6e765f40391fda37fa91c221232/src/shared/lib/comparison/ComparisonStore.ts#L1129) function. We should probably reuse that function here as well.

We may try converting `alterationEnrichmentRowData` to a `readonly` function and awaiting it here.